File: src/test/java/com/qtech/test/bubbles/TestAnimation.java
package com.qtech.test.bubbles;

import com.qtech.bubbles.graphics.Animation;

public class TestAnimation {
    public static void main(String[] args) {
        Animation animation = new Animation(100, 200, 30d);
        animation.start();

        //noinspection InfiniteLoopStatement
        while (true) {
            animation.animate();
            try {
                //noinspection BusyWait
                Thread.sleep(1000);
            } catch (InterruptedException e) {
                e.printStackTrace();
            }
        }
    }
}

File: src/test/java/com/qtech/test/bubbles/TestAnimation.kt
package com.qtech.test.bubbles

import qtech.hydro.Animation

object TestAnimation {
    @JvmStatic
    fun main(args: Array<String>) {
        val animation = Animation(100.0, 200.0, 30.0)
        animation.start()
        while (true) {
            animation.animate()
            try {
                Thread.sleep(1000)
            } catch (e: InterruptedException) {
                e.printStackTrace()
            }
        }
    }
}